adagrams/game.py

```diff
@@ -1,13 +1,117 @@
 from random import randint
 
 def draw_letters():
-    pass
+#create a list of letters
+    letters = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B', 
+    'C', 'C', 
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H', 
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J', 
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T', 
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+]
+#choose 10 letters from the list above
+#create an empty list 'hand'
+    hand = []
+#create 'for' loop to choose a letter from the list 'letters', randomly
+#'append' adds the letter to the list 'hand'
+#'pop' removes the appended letter from the list 'letters'
+    for _ in range(10):
+        random_index = randint(0,len(letters) - 1)
+        hand.append(letters[random_index]) 
+        letters.pop(random_index)
+
+    return hand
+    
+#print(draw_letters())
+#print(len(draw_letters()))
 
 def uses_available_letters(word, letter_bank):
-    pass
+#create a copy of letter_bank to avoid changing the original list of letters
+    letter_bank_copy = letter_bank[:]
+#make each letter uppercase
+    word = word.upper()
+#create a loop to remove used letter from the list if a letter was in the word
+    for letter in word:
+        if letter in letter_bank_copy:
+            letter_bank_copy.remove(letter)
+        else:
+            return False
+        
+    return True
+
+#word = "HELLO"
+#letter_bank = ["H", "E", "L", "L", "O", "A", "B", "C", "D", "E"]
+#print(uses_available_letters(word, letter_bank))
```

Comment:
Always, always remove unnecessary code that isn't needed for the final solution. In industry, commented out code should not be part of a pull request that will be reviewed because that is code that will be merged in to the main branch and released to production for customers to use. 

Also, when I'm another developer, I might wonder "What is this code for? Why is it here? Should it be uncommented so that it executes?"